spec/generator.json

```diff
@@ -31,10 +31,19 @@
             ]
         },
 
+        "source_file" {
+            "description": "Represents a source file",
+            "fields": [
+                { "name": "filename", "type": "string", "description": "The recommended name for the file, including the path to the file, where appropriate.", "example": "com/myorg/myapi/Foo.java" },
+                { "name": "source", "type": "string", "description": "The actual source code." }
```

Comment:
Should we use internet media type (https://en.wikipedia.org/wiki/Internet_media_type) for this and create a new apidoc type?